pkg/virt-controller/watch/network_test.go

```diff
@@ -163,6 +165,39 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			expectToChange,
 		),
 	)
+
+	DescribeTable("VM status interfaces requests",
+		func(ifaces []v1.Interface, ifaceRequests, expectedIfaceRequests []v1.VirtualMachineInterfaceRequest) {
+			vm := kubecli.NewMinimalVM("test")
+			vm.Spec.Template = &v1.VirtualMachineInstanceTemplateSpec{}
+			vm.Spec.Template.Spec.Domain.Devices.Interfaces = ifaces
+			vm.Status.InterfaceRequests = ifaceRequests
+
+			trimDoneInterfaceRequests(vm)
+
+			Expect(vm.Status.InterfaceRequests).To(Equal(expectedIfaceRequests))
+		},
+		Entry("have request removed on successful hotplug",
+			[]v1.Interface{{Name: "blue"}},
+			[]v1.VirtualMachineInterfaceRequest{{AddInterfaceOptions: &v1.AddInterfaceOptions{Name: "blue"}}},
+			[]v1.VirtualMachineInterfaceRequest{},
+		),
+		Entry("keep interface request for pending hotplug",
+			[]v1.Interface{},
+			[]v1.VirtualMachineInterfaceRequest{{AddInterfaceOptions: &v1.AddInterfaceOptions{Name: "blue"}}},
+			[]v1.VirtualMachineInterfaceRequest{{AddInterfaceOptions: &v1.AddInterfaceOptions{Name: "blue"}}},
+		),
+		Entry("have request removed on successful unplug",
+			[]v1.Interface{},
```

Comment:
I think you need the `absent` state, not the missing iface.

Reply:
You right, fixed